<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shop</class>
 <widget class="QWidget" name="shop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vampire Survivor</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/resourses/background.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="money">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:white;
border:5px solid rgba(196,156,89);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/resourses/money.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>10</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(45,63,197);border-radius:25px;color:white;border:10px solid rgba(196,156,89);</string>
   </property>
   <property name="text">
    <string>返 回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>524</x>
     <y>100</y>
     <width>900</width>
     <height>1090</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(76,79,113);
border:5px solid rgba(196,156,89);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buy">
   <property name="geometry">
    <rect>
     <x>1480</x>
     <y>1090</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:grey;border-radius:25px;color:white;border:10px solid rgba(196,156,89);</string>
   </property>
   <property name="text">
    <string>购 买</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid rgba(196,156,89);
image: url(:/new/prefix1/resourses/maxhp.png);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose1_1">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose1_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose1_3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>430</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1600</x>
     <y>1050</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/resourses/littlemoney.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="needmoney">
   <property name="geometry">
    <rect>
     <x>1630</x>
     <y>1050</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="choose2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>270</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid rgba(196,156,89);
image: url(:/new/prefix1/resourses/hp++.png);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="choose3">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>270</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid rgba(196,156,89);
image: url(:/new/prefix1/resourses/speed++.png);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="choose4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>550</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid rgba(196,156,89);
image: url(:/new/prefix1/resourses/interval--.png);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="choose5">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>550</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid rgba(196,156,89);
image: url(:/new/prefix1/resourses/duration++.png);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="choose6">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>550</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid rgba(196,156,89);
image: url(:/new/prefix1/resourses/collectzone++.png);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="information">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>1020</y>
     <width>871</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(139,139,139);
border:5px solid rgba(196,156,89);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose2_1">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>430</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose2_2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>430</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose2_3">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>430</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose2_4">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>430</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose2_5">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>430</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose3_1">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>430</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose3_2">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>430</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose4_1">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>710</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose4_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>710</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose5_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>710</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose5_1">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>710</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose6_2">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>710</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="choose6_1">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>710</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {width: 27px; height: 27px;}
QCheckBox{color:rgba(196,156,89);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>130</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>装备购买</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
